src/amo/components/CollectionList/styles.scss

```diff
@@ -0,0 +1,65 @@
+@import "~core/css/inc/mixins";
+@import "~photon-colors/photon-colors";
+@import "~ui/css/vars";
+
+.CollectionList {
+  padding: $padding-page;
+
+  @include respond-to(large) {
+    padding: $padding-page-l;
+  }
+
+  .Card-contents {
+    font-size: $font-size-m-smaller;
+  }
+}
+
+@include respond-to(large) {
+  .CollectionList-wrapper {
+    display: grid;
+    grid-auto-flow: column dense;
+    grid-gap: 0 $padding-page-l;
+    grid-template-columns: minmax(300px, 35%) 1fr;
+    margin: 10px 0 0;
+
+    .CollectionList-list.CardList {
+      margin: 0;
+    }
+  }
+}
+
+.CollectionList-create {
+  margin-top: 18px;
+  width: 100%;
+}
+
+.CollectionList-listing {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  margin: 0;
+  padding: 0;
+
```

Comment:
I dont really know how to leave this comment but if you had a odd number of collections, the Card is broken:

![screen shot 2018-06-25 at 14 25 38](https://user-images.githubusercontent.com/217628/41850127-c0f87f90-7883-11e8-9d77-c561baf95c29.png)

![screen shot 2018-06-25 at 14 25 35](https://user-images.githubusercontent.com/217628/41850147-ccd626dc-7883-11e8-97d2-0774ff3798ba.png)